packages/core/src/ceramic.ts

```diff
@@ -266,11 +268,17 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
       this._logger,
       params.networkOptions.name
     )
+    const anchorRequestStore = new AnchorRequestStore(
+      this._logger,
+      params.anchorLoopMinDurationMs >= 0
```

Comment:
note that `null >= 0` returns true.  We might want to clean up this condition check.